### ISSUE TITLE
Group chat "Cannot read property 'label' of undefined" (Chromium only)

### ISSUE THREAD

**Issue by tristanlins** [nextcloud/spreed#1977]
## Steps to reproduce

1. Open Nextcloud in Chromium
2. Open a group chat

### Expected behaviour

The chat history should be displayed.

### Actual behaviour

Only the list of participants can be seen.
The chat history is empty or the chat history from the previous chat is still visible.
=> Chat history is not updated to the group chat.
Instead, a JS error occurs. (see Browser log)

The error also occurs in Incognito mode. (browser apps disabled)

Works fine in Firefox.

## Browser

**Microphone available:** no

**Camera available:** no

**Operating system:** KDE neon User Edition 5.16 / Ubuntu 18.04

**Browser name:** Chromium

**Browser version:** Version 75.0.3770.90 (Offizieller Build) Built on Ubuntu , running on neon 18.04 (64-Bit)

### Browser log

<details>

```
merged.js?v=7c7acafc-6:17 Uncaught TypeError: Cannot read property 'label' of undefined
    at n._getUI (merged.js?v=7c7acafc-6:17)
    at n.getUI (merged.js?v=7c7acafc-6:17)
    at n.updateText (merged.js?v=7c7acafc-6:10843)
    at n.setLabelPlaceholder (merged.js?v=7c7acafc-6:10815)
    at n._updateNameEditability (merged.js?v=7c7acafc-6:9647)
    at n.initialize (merged.js?v=7c7acafc-6:9546)
    at n.e.View (merged.js?v=7c7acafc-6:1)
    at n.constructor (merged.js?v=7c7acafc-6:17)
    at new n (merged.js?v=7c7acafc-6:1)
    at n.updateSidebarWithActiveRoom (merged.js?v=7c7acafc-6:18401)


_getUI | @ | merged.js?v=7c7acafc-6:17
-- | -- | --
  | getUI | @ | merged.js?v=7c7acafc-6:17
  | updateText | @ | merged.js?v=7c7acafc-6:10843
  | setLabelPlaceholder | @ | merged.js?v=7c7acafc-6:10815
  | _updateNameEditability | @ | merged.js?v=7c7acafc-6:9647
  | initialize | @ | merged.js?v=7c7acafc-6:9546
  | e.View | @ | merged.js?v=7c7acafc-6:1
  | constructor | @ | merged.js?v=7c7acafc-6:17
  | n | @ | merged.js?v=7c7acafc-6:1
  | updateSidebarWithActiveRoom | @ | merged.js?v=7c7acafc-6:18401
  | (anonymous) | @ | merged.js?v=7c7acafc-6:18349
  | (anonymous) | @ | main.js?v=7c7acafc-6:39
  | u | @ | main.js?v=7c7acafc-6:39
  | fireWith | @ | main.js?v=7c7acafc-6:39
  | r.<computed> | @ | main.js?v=7c7acafc-6:39
  | success | @ | merged.js?v=7c7acafc-6:16725
  | t.success | @ | merged.js?v=7c7acafc-6:1
  | u | @ | main.js?v=7c7acafc-6:39
  | fireWith | @ | main.js?v=7c7acafc-6:39
  | k | @ | main.js?v=7c7acafc-6:39
  | (anonymous) | @ | main.js?v=7c7acafc-6:39
  | load (async) |   |  
  | send | @ | main.js?v=7c7acafc-6:39
  | ajax | @ | main.js?v=7c7acafc-6:39
  | e.ajax | @ | merged.js?v=7c7acafc-6:1
  | e.sync | @ | merged.js?v=7c7acafc-6:1
  | sync | @ | merged.js?v=7c7acafc-6:1
  | fetch | @ | merged.js?v=7c7acafc-6:1
  | OCA.Talk.Signaling.Base.syncRooms | @ | merged.js?v=7c7acafc-6:16723
  | syncAndSetActiveRoom | @ | merged.js?v=7c7acafc-6:18324
  | joinRoom | @ | merged.js?v=7c7acafc-6:17936
  | onStart | @ | merged.js?v=7c7acafc-6:18684
  | r | @ | merged.js?v=7c7acafc-6:17
  | start | @ | merged.js?v=7c7acafc-6:18
  | (anonymous) | @ | merged.js?v=7c7acafc-6:18914
  | (anonymous) | @ | merged.js?v=7c7acafc-6:18915
```

</details>

## Spreed app

**Spreed app version:** 6.0.2

**Custom TURN server configured:** no

**Custom STUN server configured:** no

## Server configuration

I'm using the official [docker image](https://hub.docker.com/_/nextcloud).

**Operating system**: Ubuntu

**Web server:** Nginx

**Database:** MariaDB 10.3.12

**PHP version:** 7.3.7

**Nextcloud Version:** 16.0.3

**List of activated apps:**

<details>

```
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - bruteforcesettings: 1.3.0
  - calendar: 1.7.0
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - contacts: 3.1.3
  - dav: 1.9.2
  - federatedfilesharing: 1.6.0
  - federation: 1.6.0
  - files: 1.11.0
  - files_pdfviewer: 1.5.0
  - files_rightclick: 0.13.0
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - firstrunwizard: 2.5.0
  - gallery: 18.3.0
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - nextcloud_announcements: 1.5.0
  - notes: 3.0.0
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - polls: 0.10.2
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - quota_warning: 1.5.0
  - recommendations: 0.4.0
  - registration: 0.4.6
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - social: 0.2.6
  - spreed: 6.0.2
  - support: 1.0.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - tasks: 0.11.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - updatenotification: 1.6.0
  - viewer: 1.0.0
  - workflowengine: 1.6.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap
```

</details>

**Nextcloud configuration:**

<details>

```
{
    "system": {
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "16.0.3.0",
        "overwrite.cli.url": "http:\/\/localhost",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_sendmailmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "default_language": "de",
        "default_locale": "de_DE",
        "maintenance": false,
        "skeletondirectory": "\/usr\/share\/nextcloud\/skeleton\/",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "loglevel": 2,
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***"
    }
}
```

</details>

**Comment by nickvergessen** [comment]
Looks like it's an error from the change name field

**Comment by tristanlins** [comment]
Here are two screenshots of the current state:

![Screenshot_20190711_163212](https://user-images.githubusercontent.com/343404/61059627-96945b00-a3f9-11e9-85e2-79f11d4c31d0.png)
![Screenshot_20190711_163203](https://user-images.githubusercontent.com/343404/61059635-98f6b500-a3f9-11e9-81a7-9756ffaa6c4a.png)

Let me know if you need more feedback 😀

**Comment by danxuliu** [comment]
@tristanlins Are you using by any chance the experimental web platform features of Chromium? Either by running it with `chrome --enable-experimental-web-platform-features` or by enabling them by visiting `chrome://flags/#enable-experimental-web-platform-features`.

**Comment by tristanlins** [comment]
![Screenshot_20190802_132114](https://user-images.githubusercontent.com/343404/62366895-6c314b80-b528-11e9-94d7-c4c0ff5bf6db.png)

Looks like 🙈
Sometime in the past I have to activate and forget this. 😬

**Comment by tristanlins** [comment]
I disabled the experimental features and now it works again. 
Should the issue remain open to correct this for future versions?

**Comment by danxuliu** [comment]
> Sometime in the past I have to activate and forget this. :grimacing:

Great that you forgot it as it allowed us to find the bug before it was widespread :-P

> Should the issue remain open to correct this for future versions?

It will be automatically closed once #2063 is merged ;-) Thanks!
